index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>Sanctuary</title>
-  <link rel="stylesheet" href="https://cdn.rawgit.com/tonsky/FiraCode/1.204/distr/fira_code.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Fira+Code:400,500,700&display=swap">
```

Comment:
As explained in [Google Fonts and font-display][1], `display=swap` sets [`font-display`][2] to `swap`.


[1]: https://css-tricks.com/google-fonts-and-font-display/
[2]: https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display